module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-646_GGTATA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-646_GGTATA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-646_GGTATA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-646_GGTATA-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-646_GGTATA-JRDRM_L001_sorted.bam > COIEggs-646_GGTATA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-646_GGTATA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-646_GGTATA-JRDRM_L001_sorted.bam | gzip > COIEggs-646_GGTATA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-646_GGTATA-JRDRM_L001_sorted.bam > COIEggs-646_GGTATA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-646_GGTATA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-646_GGTATA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-646_GGTATA-JRDRM_L001.merged.fq_sorted.bam > COIEggs-646_GGTATA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-646_GGTATA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-646_GGTATA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-646_GGTATA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-646_GGTATA-JRDRM_L001.merged.fq_sorted.bam > COIEggs-646_GGTATA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-646_GGTATA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-710_CAATTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-710_CAATTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-710_CAATTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-710_CAATTA-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-710_CAATTA-JRDRM_L001_sorted.bam > COIEggs-710_CAATTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-710_CAATTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-710_CAATTA-JRDRM_L001_sorted.bam | gzip > COIEggs-710_CAATTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-710_CAATTA-JRDRM_L001_sorted.bam > COIEggs-710_CAATTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-710_CAATTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-710_CAATTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-710_CAATTA-JRDRM_L001.merged.fq_sorted.bam > COIEggs-710_CAATTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-710_CAATTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-710_CAATTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-710_CAATTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-710_CAATTA-JRDRM_L001.merged.fq_sorted.bam > COIEggs-710_CAATTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-710_CAATTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-717_GATAGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-717_GATAGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-717_GATAGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-717_GATAGA-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-717_GATAGA-JRDRM_L001_sorted.bam > COIEggs-717_GATAGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-717_GATAGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-717_GATAGA-JRDRM_L001_sorted.bam | gzip > COIEggs-717_GATAGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-717_GATAGA-JRDRM_L001_sorted.bam > COIEggs-717_GATAGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-717_GATAGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-717_GATAGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-717_GATAGA-JRDRM_L001.merged.fq_sorted.bam > COIEggs-717_GATAGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-717_GATAGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-717_GATAGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-717_GATAGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-717_GATAGA-JRDRM_L001.merged.fq_sorted.bam > COIEggs-717_GATAGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-717_GATAGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-720_GATCTG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-720_GATCTG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-720_GATCTG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-720_GATCTG-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-720_GATCTG-JRDRM_L001_sorted.bam > COIEggs-720_GATCTG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-720_GATCTG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-720_GATCTG-JRDRM_L001_sorted.bam | gzip > COIEggs-720_GATCTG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-720_GATCTG-JRDRM_L001_sorted.bam > COIEggs-720_GATCTG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-720_GATCTG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-720_GATCTG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-720_GATCTG-JRDRM_L001.merged.fq_sorted.bam > COIEggs-720_GATCTG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-720_GATCTG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-720_GATCTG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-720_GATCTG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-720_GATCTG-JRDRM_L001.merged.fq_sorted.bam > COIEggs-720_GATCTG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-720_GATCTG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-726_TGGTCA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-726_TGGTCA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-726_TGGTCA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-726_TGGTCA-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-726_TGGTCA-JRDRM_L001_sorted.bam > COIEggs-726_TGGTCA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-726_TGGTCA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-726_TGGTCA-JRDRM_L001_sorted.bam | gzip > COIEggs-726_TGGTCA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-726_TGGTCA-JRDRM_L001_sorted.bam > COIEggs-726_TGGTCA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_sorted.bam > COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_sorted.bam > COIEggs-726_TGGTCA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-726_TGGTCA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-729_GAACAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-729_GAACAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-729_GAACAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-729_GAACAC-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-729_GAACAC-JRDRM_L001_sorted.bam > COIEggs-729_GAACAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-729_GAACAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-729_GAACAC-JRDRM_L001_sorted.bam | gzip > COIEggs-729_GAACAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-729_GAACAC-JRDRM_L001_sorted.bam > COIEggs-729_GAACAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-729_GAACAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-729_GAACAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-729_GAACAC-JRDRM_L001.merged.fq_sorted.bam > COIEggs-729_GAACAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-729_GAACAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-729_GAACAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-729_GAACAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-729_GAACAC-JRDRM_L001.merged.fq_sorted.bam > COIEggs-729_GAACAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-729_GAACAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-733_AATTCT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-733_AATTCT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-733_AATTCT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-733_AATTCT-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-733_AATTCT-JRDRM_L001_sorted.bam > COIEggs-733_AATTCT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-733_AATTCT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-733_AATTCT-JRDRM_L001_sorted.bam | gzip > COIEggs-733_AATTCT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-733_AATTCT-JRDRM_L001_sorted.bam > COIEggs-733_AATTCT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-733_AATTCT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-733_AATTCT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-733_AATTCT-JRDRM_L001.merged.fq_sorted.bam > COIEggs-733_AATTCT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-733_AATTCT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-733_AATTCT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-733_AATTCT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-733_AATTCT-JRDRM_L001.merged.fq_sorted.bam > COIEggs-733_AATTCT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-733_AATTCT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-734_CACTGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-734_CACTGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-734_CACTGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-734_CACTGT-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-734_CACTGT-JRDRM_L001_sorted.bam > COIEggs-734_CACTGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-734_CACTGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-734_CACTGT-JRDRM_L001_sorted.bam | gzip > COIEggs-734_CACTGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-734_CACTGT-JRDRM_L001_sorted.bam > COIEggs-734_CACTGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-734_CACTGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-734_CACTGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-734_CACTGT-JRDRM_L001.merged.fq_sorted.bam > COIEggs-734_CACTGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-734_CACTGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-734_CACTGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-734_CACTGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-734_CACTGT-JRDRM_L001.merged.fq_sorted.bam > COIEggs-734_CACTGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-734_CACTGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-748_AAGGTG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-748_AAGGTG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-748_AAGGTG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-748_AAGGTG-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-748_AAGGTG-JRDRM_L001_sorted.bam > COIEggs-748_AAGGTG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-748_AAGGTG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-748_AAGGTG-JRDRM_L001_sorted.bam | gzip > COIEggs-748_AAGGTG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-748_AAGGTG-JRDRM_L001_sorted.bam > COIEggs-748_AAGGTG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_sorted.bam > COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_sorted.bam > COIEggs-748_AAGGTG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-748_AAGGTG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-755_GACTCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIEggs-755_GACTCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-755_GACTCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-755_GACTCG-JRDRM_L001_sorted.bam; samtools flagstat COIEggs-755_GACTCG-JRDRM_L001_sorted.bam > COIEggs-755_GACTCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIEggs-755_GACTCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIEggs-755_GACTCG-JRDRM_L001_sorted.bam | gzip > COIEggs-755_GACTCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-755_GACTCG-JRDRM_L001_sorted.bam > COIEggs-755_GACTCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-755_GACTCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-755_GACTCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIEggs-755_GACTCG-JRDRM_L001.merged.fq_sorted.bam > COIEggs-755_GACTCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-755_GACTCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-755_GACTCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIEggs-755_GACTCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-755_GACTCG-JRDRM_L001.merged.fq_sorted.bam > COIEggs-755_GACTCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIEggs-755_GACTCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIH2O-1_GCCGCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIH2O-1_GCCGCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIH2O-1_GCCGCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIH2O-1_GCCGCG-JRDRM_L001_sorted.bam; samtools flagstat COIH2O-1_GCCGCG-JRDRM_L001_sorted.bam > COIH2O-1_GCCGCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIH2O-1_GCCGCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIH2O-1_GCCGCG-JRDRM_L001_sorted.bam | gzip > COIH2O-1_GCCGCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIH2O-1_GCCGCG-JRDRM_L001_sorted.bam > COIH2O-1_GCCGCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_sorted.bam > COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_sorted.bam > COIH2O-1_GCCGCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIH2O-1_GCCGCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIH2O_CTGGTT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIH2O_CTGGTT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIH2O_CTGGTT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIH2O_CTGGTT-JRDRM_L001_sorted.bam; samtools flagstat COIH2O_CTGGTT-JRDRM_L001_sorted.bam > COIH2O_CTGGTT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIH2O_CTGGTT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIH2O_CTGGTT-JRDRM_L001_sorted.bam | gzip > COIH2O_CTGGTT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIH2O_CTGGTT-JRDRM_L001_sorted.bam > COIH2O_CTGGTT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIH2O_CTGGTT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIH2O_CTGGTT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIH2O_CTGGTT-JRDRM_L001.merged.fq_sorted.bam > COIH2O_CTGGTT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIH2O_CTGGTT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIH2O_CTGGTT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIH2O_CTGGTT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIH2O_CTGGTT-JRDRM_L001.merged.fq_sorted.bam > COIH2O_CTGGTT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIH2O_CTGGTT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-10_GGAGGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-10_GGAGGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-10_GGAGGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-10_GGAGGT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-10_GGAGGT-JRDRM_L001_sorted.bam > COIJ0-10_GGAGGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-10_GGAGGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-10_GGAGGT-JRDRM_L001_sorted.bam | gzip > COIJ0-10_GGAGGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-10_GGAGGT-JRDRM_L001_sorted.bam > COIJ0-10_GGAGGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-10_GGAGGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-10_GGAGGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-14_CAGGCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-14_CAGGCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-14_CAGGCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-14_CAGGCG-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-14_CAGGCG-JRDRM_L001_sorted.bam > COIJ0-14_CAGGCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-14_CAGGCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-14_CAGGCG-JRDRM_L001_sorted.bam | gzip > COIJ0-14_CAGGCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-14_CAGGCG-JRDRM_L001_sorted.bam > COIJ0-14_CAGGCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-14_CAGGCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-14_CAGGCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-15_GTCATT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-15_GTCATT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-15_GTCATT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-15_GTCATT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-15_GTCATT-JRDRM_L001_sorted.bam > COIJ0-15_GTCATT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-15_GTCATT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-15_GTCATT-JRDRM_L001_sorted.bam | gzip > COIJ0-15_GTCATT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-15_GTCATT-JRDRM_L001_sorted.bam > COIJ0-15_GTCATT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-15_GTCATT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-15_GTCATT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-15_GTCATT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-15_GTCATT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-15_GTCATT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-15_GTCATT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-15_GTCATT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-15_GTCATT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-15_GTCATT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-15_GTCATT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-16_TCCATT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-16_TCCATT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-16_TCCATT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-16_TCCATT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-16_TCCATT-JRDRM_L001_sorted.bam > COIJ0-16_TCCATT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-16_TCCATT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-16_TCCATT-JRDRM_L001_sorted.bam | gzip > COIJ0-16_TCCATT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-16_TCCATT-JRDRM_L001_sorted.bam > COIJ0-16_TCCATT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-16_TCCATT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-16_TCCATT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-16_TCCATT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-16_TCCATT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-16_TCCATT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-16_TCCATT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-16_TCCATT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-16_TCCATT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-16_TCCATT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-16_TCCATT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-17_AAACAA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-17_AAACAA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-17_AAACAA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-17_AAACAA-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-17_AAACAA-JRDRM_L001_sorted.bam > COIJ0-17_AAACAA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-17_AAACAA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-17_AAACAA-JRDRM_L001_sorted.bam | gzip > COIJ0-17_AAACAA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-17_AAACAA-JRDRM_L001_sorted.bam > COIJ0-17_AAACAA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-17_AAACAA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-17_AAACAA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-17_AAACAA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-17_AAACAA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-17_AAACAA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-17_AAACAA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-17_AAACAA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-17_AAACAA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-17_AAACAA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-17_AAACAA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-1_CTACTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-1_CTACTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-1_CTACTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-1_CTACTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-1_CTACTA-JRDRM_L001_sorted.bam > COIJ0-1_CTACTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-1_CTACTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-1_CTACTA-JRDRM_L001_sorted.bam | gzip > COIJ0-1_CTACTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-1_CTACTA-JRDRM_L001_sorted.bam > COIJ0-1_CTACTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-1_CTACTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-1_CTACTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-1_CTACTA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-1_CTACTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-1_CTACTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-1_CTACTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-1_CTACTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-1_CTACTA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-1_CTACTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-1_CTACTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-21_TGCGCT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-21_TGCGCT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-21_TGCGCT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-21_TGCGCT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-21_TGCGCT-JRDRM_L001_sorted.bam > COIJ0-21_TGCGCT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-21_TGCGCT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-21_TGCGCT-JRDRM_L001_sorted.bam | gzip > COIJ0-21_TGCGCT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-21_TGCGCT-JRDRM_L001_sorted.bam > COIJ0-21_TGCGCT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-21_TGCGCT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-21_TGCGCT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-22_GCACGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-22_GCACGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-22_GCACGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-22_GCACGT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-22_GCACGT-JRDRM_L001_sorted.bam > COIJ0-22_GCACGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-22_GCACGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-22_GCACGT-JRDRM_L001_sorted.bam | gzip > COIJ0-22_GCACGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-22_GCACGT-JRDRM_L001_sorted.bam > COIJ0-22_GCACGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-22_GCACGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-22_GCACGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-22_GCACGT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-22_GCACGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-22_GCACGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-22_GCACGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-22_GCACGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-22_GCACGT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-22_GCACGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-22_GCACGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-3_CCTCCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-3_CCTCCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-3_CCTCCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-3_CCTCCC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-3_CCTCCC-JRDRM_L001_sorted.bam > COIJ0-3_CCTCCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-3_CCTCCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-3_CCTCCC-JRDRM_L001_sorted.bam | gzip > COIJ0-3_CCTCCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-3_CCTCCC-JRDRM_L001_sorted.bam > COIJ0-3_CCTCCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-3_CCTCCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-3_CCTCCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-4_TATCGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-4_TATCGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-4_TATCGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-4_TATCGA-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-4_TATCGA-JRDRM_L001_sorted.bam > COIJ0-4_TATCGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-4_TATCGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-4_TATCGA-JRDRM_L001_sorted.bam | gzip > COIJ0-4_TATCGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-4_TATCGA-JRDRM_L001_sorted.bam > COIJ0-4_TATCGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-4_TATCGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-4_TATCGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-4_TATCGA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-4_TATCGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-4_TATCGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-4_TATCGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-4_TATCGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-4_TATCGA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-4_TATCGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-4_TATCGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-683_CGATGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-683_CGATGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-683_CGATGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-683_CGATGA-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-683_CGATGA-JRDRM_L001_sorted.bam > COIJ0-683_CGATGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-683_CGATGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-683_CGATGA-JRDRM_L001_sorted.bam | gzip > COIJ0-683_CGATGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-683_CGATGA-JRDRM_L001_sorted.bam > COIJ0-683_CGATGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-683_CGATGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-683_CGATGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-683_CGATGA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-683_CGATGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-683_CGATGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-683_CGATGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-683_CGATGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-683_CGATGA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-683_CGATGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-683_CGATGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-6_CAGAAA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-6_CAGAAA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-6_CAGAAA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-6_CAGAAA-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-6_CAGAAA-JRDRM_L001_sorted.bam > COIJ0-6_CAGAAA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-6_CAGAAA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-6_CAGAAA-JRDRM_L001_sorted.bam | gzip > COIJ0-6_CAGAAA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-6_CAGAAA-JRDRM_L001_sorted.bam > COIJ0-6_CAGAAA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-6_CAGAAA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-6_CAGAAA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-725_AGTGTC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-725_AGTGTC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-725_AGTGTC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-725_AGTGTC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-725_AGTGTC-JRDRM_L001_sorted.bam > COIJ0-725_AGTGTC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-725_AGTGTC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-725_AGTGTC-JRDRM_L001_sorted.bam | gzip > COIJ0-725_AGTGTC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-725_AGTGTC-JRDRM_L001_sorted.bam > COIJ0-725_AGTGTC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-725_AGTGTC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-725_AGTGTC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-729_CCGTAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-729_CCGTAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-729_CCGTAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-729_CCGTAG-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-729_CCGTAG-JRDRM_L001_sorted.bam > COIJ0-729_CCGTAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-729_CCGTAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-729_CCGTAG-JRDRM_L001_sorted.bam | gzip > COIJ0-729_CCGTAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-729_CCGTAG-JRDRM_L001_sorted.bam > COIJ0-729_CCGTAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-729_CCGTAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-729_CCGTAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-733_GCCCGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-733_GCCCGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-733_GCCCGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-733_GCCCGA-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-733_GCCCGA-JRDRM_L001_sorted.bam > COIJ0-733_GCCCGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-733_GCCCGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-733_GCCCGA-JRDRM_L001_sorted.bam | gzip > COIJ0-733_GCCCGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-733_GCCCGA-JRDRM_L001_sorted.bam > COIJ0-733_GCCCGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-733_GCCCGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-733_GCCCGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-734_TAGCAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-734_TAGCAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-734_TAGCAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-734_TAGCAG-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-734_TAGCAG-JRDRM_L001_sorted.bam > COIJ0-734_TAGCAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-734_TAGCAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-734_TAGCAG-JRDRM_L001_sorted.bam | gzip > COIJ0-734_TAGCAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-734_TAGCAG-JRDRM_L001_sorted.bam > COIJ0-734_TAGCAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-734_TAGCAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-734_TAGCAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-735_ATTGAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-735_ATTGAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-735_ATTGAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-735_ATTGAG-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-735_ATTGAG-JRDRM_L001_sorted.bam > COIJ0-735_ATTGAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-735_ATTGAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-735_ATTGAG-JRDRM_L001_sorted.bam | gzip > COIJ0-735_ATTGAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-735_ATTGAG-JRDRM_L001_sorted.bam > COIJ0-735_ATTGAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-735_ATTGAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-735_ATTGAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-739_AGGATA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-739_AGGATA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-739_AGGATA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-739_AGGATA-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-739_AGGATA-JRDRM_L001_sorted.bam > COIJ0-739_AGGATA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-739_AGGATA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-739_AGGATA-JRDRM_L001_sorted.bam | gzip > COIJ0-739_AGGATA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-739_AGGATA-JRDRM_L001_sorted.bam > COIJ0-739_AGGATA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-739_AGGATA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-739_AGGATA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-739_AGGATA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-739_AGGATA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-739_AGGATA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-739_AGGATA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-739_AGGATA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-739_AGGATA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-739_AGGATA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-739_AGGATA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-742_CGCATT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-742_CGCATT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-742_CGCATT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-742_CGCATT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-742_CGCATT-JRDRM_L001_sorted.bam > COIJ0-742_CGCATT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-742_CGCATT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-742_CGCATT-JRDRM_L001_sorted.bam | gzip > COIJ0-742_CGCATT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-742_CGCATT-JRDRM_L001_sorted.bam > COIJ0-742_CGCATT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-742_CGCATT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-742_CGCATT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-742_CGCATT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-742_CGCATT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-742_CGCATT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-742_CGCATT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-742_CGCATT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-742_CGCATT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-742_CGCATT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-742_CGCATT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-743_TGCTCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-743_TGCTCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-743_TGCTCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-743_TGCTCC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-743_TGCTCC-JRDRM_L001_sorted.bam > COIJ0-743_TGCTCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-743_TGCTCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-743_TGCTCC-JRDRM_L001_sorted.bam | gzip > COIJ0-743_TGCTCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-743_TGCTCC-JRDRM_L001_sorted.bam > COIJ0-743_TGCTCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-743_TGCTCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-743_TGCTCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-744_GTCAGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-744_GTCAGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-744_GTCAGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-744_GTCAGG-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-744_GTCAGG-JRDRM_L001_sorted.bam > COIJ0-744_GTCAGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-744_GTCAGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-744_GTCAGG-JRDRM_L001_sorted.bam | gzip > COIJ0-744_GTCAGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-744_GTCAGG-JRDRM_L001_sorted.bam > COIJ0-744_GTCAGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-744_GTCAGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-744_GTCAGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-745_TCTGGC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-745_TCTGGC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-745_TCTGGC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-745_TCTGGC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-745_TCTGGC-JRDRM_L001_sorted.bam > COIJ0-745_TCTGGC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-745_TCTGGC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-745_TCTGGC-JRDRM_L001_sorted.bam | gzip > COIJ0-745_TCTGGC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-745_TCTGGC-JRDRM_L001_sorted.bam > COIJ0-745_TCTGGC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-745_TCTGGC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-745_TCTGGC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-746_AATAAT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-746_AATAAT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-746_AATAAT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-746_AATAAT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-746_AATAAT-JRDRM_L001_sorted.bam > COIJ0-746_AATAAT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-746_AATAAT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-746_AATAAT-JRDRM_L001_sorted.bam | gzip > COIJ0-746_AATAAT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-746_AATAAT-JRDRM_L001_sorted.bam > COIJ0-746_AATAAT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-746_AATAAT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-746_AATAAT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-746_AATAAT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-746_AATAAT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-746_AATAAT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-746_AATAAT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-746_AATAAT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-746_AATAAT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-746_AATAAT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-746_AATAAT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-747_ACGTCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-747_ACGTCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-747_ACGTCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-747_ACGTCG-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-747_ACGTCG-JRDRM_L001_sorted.bam > COIJ0-747_ACGTCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-747_ACGTCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-747_ACGTCG-JRDRM_L001_sorted.bam | gzip > COIJ0-747_ACGTCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-747_ACGTCG-JRDRM_L001_sorted.bam > COIJ0-747_ACGTCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-747_ACGTCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-747_ACGTCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-748_GATTTC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-748_GATTTC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-748_GATTTC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-748_GATTTC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-748_GATTTC-JRDRM_L001_sorted.bam > COIJ0-748_GATTTC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-748_GATTTC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-748_GATTTC-JRDRM_L001_sorted.bam | gzip > COIJ0-748_GATTTC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-748_GATTTC-JRDRM_L001_sorted.bam > COIJ0-748_GATTTC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-748_GATTTC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-748_GATTTC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-748_GATTTC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-748_GATTTC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-748_GATTTC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-748_GATTTC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-748_GATTTC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-748_GATTTC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-748_GATTTC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-748_GATTTC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-749_CTAAAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-749_CTAAAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-749_CTAAAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-749_CTAAAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-749_CTAAAC-JRDRM_L001_sorted.bam > COIJ0-749_CTAAAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-749_CTAAAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-749_CTAAAC-JRDRM_L001_sorted.bam | gzip > COIJ0-749_CTAAAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-749_CTAAAC-JRDRM_L001_sorted.bam > COIJ0-749_CTAAAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-749_CTAAAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-749_CTAAAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-750_CGCCTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-750_CGCCTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-750_CGCCTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-750_CGCCTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-750_CGCCTA-JRDRM_L001_sorted.bam > COIJ0-750_CGCCTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-750_CGCCTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-750_CGCCTA-JRDRM_L001_sorted.bam | gzip > COIJ0-750_CGCCTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-750_CGCCTA-JRDRM_L001_sorted.bam > COIJ0-750_CGCCTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_sorted.bam > COIJ0-750_CGCCTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-750_CGCCTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-752_ATATCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-752_ATATCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-752_ATATCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-752_ATATCC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-752_ATATCC-JRDRM_L001_sorted.bam > COIJ0-752_ATATCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-752_ATATCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-752_ATATCC-JRDRM_L001_sorted.bam | gzip > COIJ0-752_ATATCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-752_ATATCC-JRDRM_L001_sorted.bam > COIJ0-752_ATATCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-752_ATATCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-752_ATATCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-752_ATATCC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-752_ATATCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-752_ATATCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-752_ATATCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-752_ATATCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-752_ATATCC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-752_ATATCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-752_ATATCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-754_TACACC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-754_TACACC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-754_TACACC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-754_TACACC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-754_TACACC-JRDRM_L001_sorted.bam > COIJ0-754_TACACC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-754_TACACC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-754_TACACC-JRDRM_L001_sorted.bam | gzip > COIJ0-754_TACACC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-754_TACACC-JRDRM_L001_sorted.bam > COIJ0-754_TACACC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-754_TACACC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-754_TACACC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-754_TACACC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-754_TACACC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-754_TACACC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-754_TACACC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-754_TACACC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-754_TACACC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-754_TACACC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-754_TACACC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-755_GAATCT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-755_GAATCT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-755_GAATCT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-755_GAATCT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-755_GAATCT-JRDRM_L001_sorted.bam > COIJ0-755_GAATCT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-755_GAATCT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-755_GAATCT-JRDRM_L001_sorted.bam | gzip > COIJ0-755_GAATCT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-755_GAATCT-JRDRM_L001_sorted.bam > COIJ0-755_GAATCT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-755_GAATCT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-755_GAATCT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-755_GAATCT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-755_GAATCT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-755_GAATCT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-755_GAATCT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-755_GAATCT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-755_GAATCT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-755_GAATCT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-755_GAATCT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-7_TCTGAT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-7_TCTGAT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-7_TCTGAT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-7_TCTGAT-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-7_TCTGAT-JRDRM_L001_sorted.bam > COIJ0-7_TCTGAT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-7_TCTGAT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-7_TCTGAT-JRDRM_L001_sorted.bam | gzip > COIJ0-7_TCTGAT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-7_TCTGAT-JRDRM_L001_sorted.bam > COIJ0-7_TCTGAT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_sorted.bam > COIJ0-7_TCTGAT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-7_TCTGAT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-8_CAGTTC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-8_CAGTTC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-8_CAGTTC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-8_CAGTTC-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-8_CAGTTC-JRDRM_L001_sorted.bam > COIJ0-8_CAGTTC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-8_CAGTTC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-8_CAGTTC-JRDRM_L001_sorted.bam | gzip > COIJ0-8_CAGTTC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-8_CAGTTC-JRDRM_L001_sorted.bam > COIJ0-8_CAGTTC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_sorted.bam > COIJ0-8_CAGTTC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-8_CAGTTC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-9_GGTCCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ0-9_GGTCCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-9_GGTCCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-9_GGTCCG-JRDRM_L001_sorted.bam; samtools flagstat COIJ0-9_GGTCCG-JRDRM_L001_sorted.bam > COIJ0-9_GGTCCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ0-9_GGTCCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ0-9_GGTCCG-JRDRM_L001_sorted.bam | gzip > COIJ0-9_GGTCCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-9_GGTCCG-JRDRM_L001_sorted.bam > COIJ0-9_GGTCCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_sorted.bam > COIJ0-9_GGTCCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ0-9_GGTCCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-10_CACCGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-10_CACCGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-10_CACCGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-10_CACCGG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-10_CACCGG-JRDRM_L001_sorted.bam > COIJ28-10_CACCGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-10_CACCGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-10_CACCGG-JRDRM_L001_sorted.bam | gzip > COIJ28-10_CACCGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-10_CACCGG-JRDRM_L001_sorted.bam > COIJ28-10_CACCGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-10_CACCGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-10_CACCGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-10_CACCGG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-10_CACCGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-10_CACCGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-10_CACCGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-10_CACCGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-10_CACCGG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-10_CACCGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-10_CACCGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-12_CTTGGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-12_CTTGGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-12_CTTGGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-12_CTTGGG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-12_CTTGGG-JRDRM_L001_sorted.bam > COIJ28-12_CTTGGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-12_CTTGGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-12_CTTGGG-JRDRM_L001_sorted.bam | gzip > COIJ28-12_CTTGGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-12_CTTGGG-JRDRM_L001_sorted.bam > COIJ28-12_CTTGGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-12_CTTGGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-12_CTTGGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-14_CGCACA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-14_CGCACA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-14_CGCACA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-14_CGCACA-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-14_CGCACA-JRDRM_L001_sorted.bam > COIJ28-14_CGCACA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-14_CGCACA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-14_CGCACA-JRDRM_L001_sorted.bam | gzip > COIJ28-14_CGCACA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-14_CGCACA-JRDRM_L001_sorted.bam > COIJ28-14_CGCACA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-14_CGCACA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-14_CGCACA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-14_CGCACA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-14_CGCACA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-14_CGCACA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-14_CGCACA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-14_CGCACA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-14_CGCACA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-14_CGCACA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-14_CGCACA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-15_TGCCAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-15_TGCCAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-15_TGCCAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-15_TGCCAG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-15_TGCCAG-JRDRM_L001_sorted.bam > COIJ28-15_TGCCAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-15_TGCCAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-15_TGCCAG-JRDRM_L001_sorted.bam | gzip > COIJ28-15_TGCCAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-15_TGCCAG-JRDRM_L001_sorted.bam > COIJ28-15_TGCCAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-15_TGCCAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-15_TGCCAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-17_CAACGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-17_CAACGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-17_CAACGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-17_CAACGA-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-17_CAACGA-JRDRM_L001_sorted.bam > COIJ28-17_CAACGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-17_CAACGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-17_CAACGA-JRDRM_L001_sorted.bam | gzip > COIJ28-17_CAACGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-17_CAACGA-JRDRM_L001_sorted.bam > COIJ28-17_CAACGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-17_CAACGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-17_CAACGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-17_CAACGA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-17_CAACGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-17_CAACGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-17_CAACGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-17_CAACGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-17_CAACGA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-17_CAACGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-17_CAACGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-19_TTTCCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-19_TTTCCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-19_TTTCCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-19_TTTCCG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-19_TTTCCG-JRDRM_L001_sorted.bam > COIJ28-19_TTTCCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-19_TTTCCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-19_TTTCCG-JRDRM_L001_sorted.bam | gzip > COIJ28-19_TTTCCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-19_TTTCCG-JRDRM_L001_sorted.bam > COIJ28-19_TTTCCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-19_TTTCCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-19_TTTCCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-1_CTGCAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-1_CTGCAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-1_CTGCAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-1_CTGCAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-1_CTGCAC-JRDRM_L001_sorted.bam > COIJ28-1_CTGCAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-1_CTGCAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-1_CTGCAC-JRDRM_L001_sorted.bam | gzip > COIJ28-1_CTGCAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-1_CTGCAC-JRDRM_L001_sorted.bam > COIJ28-1_CTGCAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-1_CTGCAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-1_CTGCAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-21_ACACCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-21_ACACCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-21_ACACCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-21_ACACCG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-21_ACACCG-JRDRM_L001_sorted.bam > COIJ28-21_ACACCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-21_ACACCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-21_ACACCG-JRDRM_L001_sorted.bam | gzip > COIJ28-21_ACACCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-21_ACACCG-JRDRM_L001_sorted.bam > COIJ28-21_ACACCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-21_ACACCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-21_ACACCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-21_ACACCG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-21_ACACCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-21_ACACCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-21_ACACCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-21_ACACCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-21_ACACCG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-21_ACACCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-21_ACACCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-22_AGTTGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-22_AGTTGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-22_AGTTGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-22_AGTTGA-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-22_AGTTGA-JRDRM_L001_sorted.bam > COIJ28-22_AGTTGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-22_AGTTGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-22_AGTTGA-JRDRM_L001_sorted.bam | gzip > COIJ28-22_AGTTGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-22_AGTTGA-JRDRM_L001_sorted.bam > COIJ28-22_AGTTGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-22_AGTTGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-22_AGTTGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-2_ACGGTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-2_ACGGTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-2_ACGGTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-2_ACGGTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-2_ACGGTA-JRDRM_L001_sorted.bam > COIJ28-2_ACGGTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-2_ACGGTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-2_ACGGTA-JRDRM_L001_sorted.bam | gzip > COIJ28-2_ACGGTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-2_ACGGTA-JRDRM_L001_sorted.bam > COIJ28-2_ACGGTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-2_ACGGTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-2_ACGGTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-4_TAAGAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-4_TAAGAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-4_TAAGAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-4_TAAGAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-4_TAAGAC-JRDRM_L001_sorted.bam > COIJ28-4_TAAGAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-4_TAAGAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-4_TAAGAC-JRDRM_L001_sorted.bam | gzip > COIJ28-4_TAAGAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-4_TAAGAC-JRDRM_L001_sorted.bam > COIJ28-4_TAAGAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-4_TAAGAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-4_TAAGAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-5_CCCGAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-5_CCCGAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-5_CCCGAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-5_CCCGAG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-5_CCCGAG-JRDRM_L001_sorted.bam > COIJ28-5_CCCGAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-5_CCCGAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-5_CCCGAG-JRDRM_L001_sorted.bam | gzip > COIJ28-5_CCCGAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-5_CCCGAG-JRDRM_L001_sorted.bam > COIJ28-5_CCCGAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-5_CCCGAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-5_CCCGAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-6_GCAATA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-6_GCAATA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-6_GCAATA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-6_GCAATA-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-6_GCAATA-JRDRM_L001_sorted.bam > COIJ28-6_GCAATA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-6_GCAATA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-6_GCAATA-JRDRM_L001_sorted.bam | gzip > COIJ28-6_GCAATA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-6_GCAATA-JRDRM_L001_sorted.bam > COIJ28-6_GCAATA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-6_GCAATA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-6_GCAATA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-6_GCAATA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-6_GCAATA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-6_GCAATA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-6_GCAATA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-6_GCAATA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-6_GCAATA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-6_GCAATA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-6_GCAATA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-729_CTATGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-729_CTATGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-729_CTATGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-729_CTATGG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-729_CTATGG-JRDRM_L001_sorted.bam > COIJ28-729_CTATGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-729_CTATGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-729_CTATGG-JRDRM_L001_sorted.bam | gzip > COIJ28-729_CTATGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-729_CTATGG-JRDRM_L001_sorted.bam > COIJ28-729_CTATGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-729_CTATGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-729_CTATGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-729_CTATGG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-729_CTATGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-729_CTATGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-729_CTATGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-729_CTATGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-729_CTATGG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-729_CTATGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-729_CTATGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-733_TATCCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-733_TATCCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-733_TATCCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-733_TATCCC-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-733_TATCCC-JRDRM_L001_sorted.bam > COIJ28-733_TATCCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-733_TATCCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-733_TATCCC-JRDRM_L001_sorted.bam | gzip > COIJ28-733_TATCCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-733_TATCCC-JRDRM_L001_sorted.bam > COIJ28-733_TATCCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-733_TATCCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-733_TATCCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-733_TATCCC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-733_TATCCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-733_TATCCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-733_TATCCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-733_TATCCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-733_TATCCC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-733_TATCCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-733_TATCCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-734_GCCCTT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-734_GCCCTT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-734_GCCCTT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-734_GCCCTT-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-734_GCCCTT-JRDRM_L001_sorted.bam > COIJ28-734_GCCCTT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-734_GCCCTT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-734_GCCCTT-JRDRM_L001_sorted.bam | gzip > COIJ28-734_GCCCTT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-734_GCCCTT-JRDRM_L001_sorted.bam > COIJ28-734_GCCCTT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_sorted.bam > COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_sorted.bam > COIJ28-734_GCCCTT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-734_GCCCTT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-739_CGGTTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-739_CGGTTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-739_CGGTTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-739_CGGTTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-739_CGGTTA-JRDRM_L001_sorted.bam > COIJ28-739_CGGTTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-739_CGGTTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-739_CGGTTA-JRDRM_L001_sorted.bam | gzip > COIJ28-739_CGGTTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-739_CGGTTA-JRDRM_L001_sorted.bam > COIJ28-739_CGGTTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-739_CGGTTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-739_CGGTTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-742_CCCACG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-742_CCCACG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-742_CCCACG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-742_CCCACG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-742_CCCACG-JRDRM_L001_sorted.bam > COIJ28-742_CCCACG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-742_CCCACG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-742_CCCACG-JRDRM_L001_sorted.bam | gzip > COIJ28-742_CCCACG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-742_CCCACG-JRDRM_L001_sorted.bam > COIJ28-742_CCCACG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-742_CCCACG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-742_CCCACG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-742_CCCACG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-742_CCCACG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-742_CCCACG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-742_CCCACG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-742_CCCACG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-742_CCCACG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-742_CCCACG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-742_CCCACG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-743_TTATAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-743_TTATAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-743_TTATAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-743_TTATAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-743_TTATAC-JRDRM_L001_sorted.bam > COIJ28-743_TTATAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-743_TTATAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-743_TTATAC-JRDRM_L001_sorted.bam | gzip > COIJ28-743_TTATAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-743_TTATAC-JRDRM_L001_sorted.bam > COIJ28-743_TTATAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-743_TTATAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-743_TTATAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-743_TTATAC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-743_TTATAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-743_TTATAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-743_TTATAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-743_TTATAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-743_TTATAC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-743_TTATAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-743_TTATAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-745_CATAAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-745_CATAAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-745_CATAAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-745_CATAAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-745_CATAAC-JRDRM_L001_sorted.bam > COIJ28-745_CATAAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-745_CATAAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-745_CATAAC-JRDRM_L001_sorted.bam | gzip > COIJ28-745_CATAAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-745_CATAAC-JRDRM_L001_sorted.bam > COIJ28-745_CATAAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-745_CATAAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-745_CATAAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-745_CATAAC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-745_CATAAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-745_CATAAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-745_CATAAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-745_CATAAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-745_CATAAC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-745_CATAAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-745_CATAAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-748_GTAGCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-748_GTAGCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-748_GTAGCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-748_GTAGCC-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-748_GTAGCC-JRDRM_L001_sorted.bam > COIJ28-748_GTAGCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-748_GTAGCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-748_GTAGCC-JRDRM_L001_sorted.bam | gzip > COIJ28-748_GTAGCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-748_GTAGCC-JRDRM_L001_sorted.bam > COIJ28-748_GTAGCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-748_GTAGCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-748_GTAGCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-749_ATAGGC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-749_ATAGGC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-749_ATAGGC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-749_ATAGGC-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-749_ATAGGC-JRDRM_L001_sorted.bam > COIJ28-749_ATAGGC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-749_ATAGGC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-749_ATAGGC-JRDRM_L001_sorted.bam | gzip > COIJ28-749_ATAGGC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-749_ATAGGC-JRDRM_L001_sorted.bam > COIJ28-749_ATAGGC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-749_ATAGGC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-749_ATAGGC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-750_GGCTGC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-750_GGCTGC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-750_GGCTGC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-750_GGCTGC-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-750_GGCTGC-JRDRM_L001_sorted.bam > COIJ28-750_GGCTGC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-750_GGCTGC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-750_GGCTGC-JRDRM_L001_sorted.bam | gzip > COIJ28-750_GGCTGC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-750_GGCTGC-JRDRM_L001_sorted.bam > COIJ28-750_GGCTGC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_sorted.bam > COIJ28-750_GGCTGC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-750_GGCTGC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-753_CGATCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-753_CGATCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-753_CGATCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-753_CGATCG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-753_CGATCG-JRDRM_L001_sorted.bam > COIJ28-753_CGATCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-753_CGATCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-753_CGATCG-JRDRM_L001_sorted.bam | gzip > COIJ28-753_CGATCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-753_CGATCG-JRDRM_L001_sorted.bam > COIJ28-753_CGATCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-753_CGATCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-753_CGATCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-753_CGATCG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-753_CGATCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-753_CGATCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-753_CGATCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-753_CGATCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-753_CGATCG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-753_CGATCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-753_CGATCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-755_AGGAGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-755_AGGAGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-755_AGGAGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-755_AGGAGG-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-755_AGGAGG-JRDRM_L001_sorted.bam > COIJ28-755_AGGAGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-755_AGGAGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-755_AGGAGG-JRDRM_L001_sorted.bam | gzip > COIJ28-755_AGGAGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-755_AGGAGG-JRDRM_L001_sorted.bam > COIJ28-755_AGGAGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_sorted.bam > COIJ28-755_AGGAGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-755_AGGAGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-8_CGCTCT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-8_CGCTCT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-8_CGCTCT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-8_CGCTCT-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-8_CGCTCT-JRDRM_L001_sorted.bam > COIJ28-8_CGCTCT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-8_CGCTCT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-8_CGCTCT-JRDRM_L001_sorted.bam | gzip > COIJ28-8_CGCTCT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-8_CGCTCT-JRDRM_L001_sorted.bam > COIJ28-8_CGCTCT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_sorted.bam > COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_sorted.bam > COIJ28-8_CGCTCT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-8_CGCTCT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-9_TCCAGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ28-9_TCCAGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-9_TCCAGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-9_TCCAGA-JRDRM_L001_sorted.bam; samtools flagstat COIJ28-9_TCCAGA-JRDRM_L001_sorted.bam > COIJ28-9_TCCAGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ28-9_TCCAGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ28-9_TCCAGA-JRDRM_L001_sorted.bam | gzip > COIJ28-9_TCCAGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-9_TCCAGA-JRDRM_L001_sorted.bam > COIJ28-9_TCCAGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_sorted.bam > COIJ28-9_TCCAGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ28-9_TCCAGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-729_CTCCCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-729_CTCCCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-729_CTCCCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-729_CTCCCC-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-729_CTCCCC-JRDRM_L001_sorted.bam > COIJ35-729_CTCCCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-729_CTCCCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-729_CTCCCC-JRDRM_L001_sorted.bam | gzip > COIJ35-729_CTCCCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-729_CTCCCC-JRDRM_L001_sorted.bam > COIJ35-729_CTCCCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-729_CTCCCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-729_CTCCCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-733_GATGAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-733_GATGAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-733_GATGAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-733_GATGAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-733_GATGAC-JRDRM_L001_sorted.bam > COIJ35-733_GATGAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-733_GATGAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-733_GATGAC-JRDRM_L001_sorted.bam | gzip > COIJ35-733_GATGAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-733_GATGAC-JRDRM_L001_sorted.bam > COIJ35-733_GATGAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-733_GATGAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-733_GATGAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-733_GATGAC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-733_GATGAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-733_GATGAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-733_GATGAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-733_GATGAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-733_GATGAC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-733_GATGAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-733_GATGAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-734_ATACAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-734_ATACAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-734_ATACAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-734_ATACAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-734_ATACAC-JRDRM_L001_sorted.bam > COIJ35-734_ATACAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-734_ATACAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-734_ATACAC-JRDRM_L001_sorted.bam | gzip > COIJ35-734_ATACAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-734_ATACAC-JRDRM_L001_sorted.bam > COIJ35-734_ATACAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-734_ATACAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-734_ATACAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-734_ATACAC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-734_ATACAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-734_ATACAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-734_ATACAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-734_ATACAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-734_ATACAC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-734_ATACAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-734_ATACAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-735_TATTGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-735_TATTGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-735_TATTGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-735_TATTGG-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-735_TATTGG-JRDRM_L001_sorted.bam > COIJ35-735_TATTGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-735_TATTGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-735_TATTGG-JRDRM_L001_sorted.bam | gzip > COIJ35-735_TATTGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-735_TATTGG-JRDRM_L001_sorted.bam > COIJ35-735_TATTGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-735_TATTGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-735_TATTGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-735_TATTGG-JRDRM_L001.merged.fq_sorted.bam > COIJ35-735_TATTGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-735_TATTGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-735_TATTGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-735_TATTGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-735_TATTGG-JRDRM_L001.merged.fq_sorted.bam > COIJ35-735_TATTGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-735_TATTGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-739_CGTGAT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-739_CGTGAT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-739_CGTGAT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-739_CGTGAT-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-739_CGTGAT-JRDRM_L001_sorted.bam > COIJ35-739_CGTGAT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-739_CGTGAT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-739_CGTGAT-JRDRM_L001_sorted.bam | gzip > COIJ35-739_CGTGAT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-739_CGTGAT-JRDRM_L001_sorted.bam > COIJ35-739_CGTGAT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_sorted.bam > COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_sorted.bam > COIJ35-739_CGTGAT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-739_CGTGAT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-742_ACGGGC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-742_ACGGGC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-742_ACGGGC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-742_ACGGGC-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-742_ACGGGC-JRDRM_L001_sorted.bam > COIJ35-742_ACGGGC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-742_ACGGGC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-742_ACGGGC-JRDRM_L001_sorted.bam | gzip > COIJ35-742_ACGGGC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-742_ACGGGC-JRDRM_L001_sorted.bam > COIJ35-742_ACGGGC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-742_ACGGGC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-742_ACGGGC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-743_GCGGTT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-743_GCGGTT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-743_GCGGTT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-743_GCGGTT-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-743_GCGGTT-JRDRM_L001_sorted.bam > COIJ35-743_GCGGTT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-743_GCGGTT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-743_GCGGTT-JRDRM_L001_sorted.bam | gzip > COIJ35-743_GCGGTT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-743_GCGGTT-JRDRM_L001_sorted.bam > COIJ35-743_GCGGTT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_sorted.bam > COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_sorted.bam > COIJ35-743_GCGGTT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-743_GCGGTT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-744_CACGGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-744_CACGGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-744_CACGGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-744_CACGGA-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-744_CACGGA-JRDRM_L001_sorted.bam > COIJ35-744_CACGGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-744_CACGGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-744_CACGGA-JRDRM_L001_sorted.bam | gzip > COIJ35-744_CACGGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-744_CACGGA-JRDRM_L001_sorted.bam > COIJ35-744_CACGGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-744_CACGGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-744_CACGGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-744_CACGGA-JRDRM_L001.merged.fq_sorted.bam > COIJ35-744_CACGGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-744_CACGGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-744_CACGGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-744_CACGGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-744_CACGGA-JRDRM_L001.merged.fq_sorted.bam > COIJ35-744_CACGGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-744_CACGGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-748_CCTGTC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-748_CCTGTC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-748_CCTGTC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-748_CCTGTC-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-748_CCTGTC-JRDRM_L001_sorted.bam > COIJ35-748_CCTGTC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-748_CCTGTC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-748_CCTGTC-JRDRM_L001_sorted.bam | gzip > COIJ35-748_CCTGTC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-748_CCTGTC-JRDRM_L001_sorted.bam > COIJ35-748_CCTGTC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_sorted.bam > COIJ35-748_CCTGTC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-748_CCTGTC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-749_ATTTTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-749_ATTTTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-749_ATTTTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-749_ATTTTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-749_ATTTTA-JRDRM_L001_sorted.bam > COIJ35-749_ATTTTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-749_ATTTTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-749_ATTTTA-JRDRM_L001_sorted.bam | gzip > COIJ35-749_ATTTTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-749_ATTTTA-JRDRM_L001_sorted.bam > COIJ35-749_ATTTTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_sorted.bam > COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_sorted.bam > COIJ35-749_ATTTTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-749_ATTTTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-750_TGTCTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-750_TGTCTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-750_TGTCTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-750_TGTCTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-750_TGTCTA-JRDRM_L001_sorted.bam > COIJ35-750_TGTCTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-750_TGTCTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-750_TGTCTA-JRDRM_L001_sorted.bam | gzip > COIJ35-750_TGTCTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-750_TGTCTA-JRDRM_L001_sorted.bam > COIJ35-750_TGTCTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_sorted.bam > COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_sorted.bam > COIJ35-750_TGTCTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-750_TGTCTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-754_GGGCGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ35-754_GGGCGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-754_GGGCGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-754_GGGCGT-JRDRM_L001_sorted.bam; samtools flagstat COIJ35-754_GGGCGT-JRDRM_L001_sorted.bam > COIJ35-754_GGGCGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ35-754_GGGCGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ35-754_GGGCGT-JRDRM_L001_sorted.bam | gzip > COIJ35-754_GGGCGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-754_GGGCGT-JRDRM_L001_sorted.bam > COIJ35-754_GGGCGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_sorted.bam > COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_sorted.bam > COIJ35-754_GGGCGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ35-754_GGGCGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-683_TCAATC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-683_TCAATC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-683_TCAATC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-683_TCAATC-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-683_TCAATC-JRDRM_L001_sorted.bam > COIJ42-683_TCAATC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-683_TCAATC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-683_TCAATC-JRDRM_L001_sorted.bam | gzip > COIJ42-683_TCAATC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-683_TCAATC-JRDRM_L001_sorted.bam > COIJ42-683_TCAATC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-683_TCAATC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-683_TCAATC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-683_TCAATC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-683_TCAATC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-683_TCAATC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-683_TCAATC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-683_TCAATC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-683_TCAATC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-683_TCAATC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-683_TCAATC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-729_GACTAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-729_GACTAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-729_GACTAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-729_GACTAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-729_GACTAC-JRDRM_L001_sorted.bam > COIJ42-729_GACTAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-729_GACTAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-729_GACTAC-JRDRM_L001_sorted.bam | gzip > COIJ42-729_GACTAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-729_GACTAC-JRDRM_L001_sorted.bam > COIJ42-729_GACTAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-729_GACTAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-729_GACTAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-729_GACTAC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-729_GACTAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-729_GACTAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-729_GACTAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-729_GACTAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-729_GACTAC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-729_GACTAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-729_GACTAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-733_ACCGAA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-733_ACCGAA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-733_ACCGAA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-733_ACCGAA-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-733_ACCGAA-JRDRM_L001_sorted.bam > COIJ42-733_ACCGAA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-733_ACCGAA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-733_ACCGAA-JRDRM_L001_sorted.bam | gzip > COIJ42-733_ACCGAA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-733_ACCGAA-JRDRM_L001_sorted.bam > COIJ42-733_ACCGAA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_sorted.bam > COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_sorted.bam > COIJ42-733_ACCGAA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-733_ACCGAA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-734_TCGTAT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-734_TCGTAT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-734_TCGTAT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-734_TCGTAT-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-734_TCGTAT-JRDRM_L001_sorted.bam > COIJ42-734_TCGTAT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-734_TCGTAT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-734_TCGTAT-JRDRM_L001_sorted.bam | gzip > COIJ42-734_TCGTAT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-734_TCGTAT-JRDRM_L001_sorted.bam > COIJ42-734_TCGTAT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_sorted.bam > COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_sorted.bam > COIJ42-734_TCGTAT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-734_TCGTAT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-735_GGCGTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-735_GGCGTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-735_GGCGTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-735_GGCGTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-735_GGCGTA-JRDRM_L001_sorted.bam > COIJ42-735_GGCGTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-735_GGCGTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-735_GGCGTA-JRDRM_L001_sorted.bam | gzip > COIJ42-735_GGCGTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-735_GGCGTA-JRDRM_L001_sorted.bam > COIJ42-735_GGCGTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_sorted.bam > COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_sorted.bam > COIJ42-735_GGCGTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-735_GGCGTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-739_TGCGTC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-739_TGCGTC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-739_TGCGTC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-739_TGCGTC-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-739_TGCGTC-JRDRM_L001_sorted.bam > COIJ42-739_TGCGTC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-739_TGCGTC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-739_TGCGTC-JRDRM_L001_sorted.bam | gzip > COIJ42-739_TGCGTC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-739_TGCGTC-JRDRM_L001_sorted.bam > COIJ42-739_TGCGTC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-739_TGCGTC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-739_TGCGTC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-742_AGACGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-742_AGACGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-742_AGACGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-742_AGACGA-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-742_AGACGA-JRDRM_L001_sorted.bam > COIJ42-742_AGACGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-742_AGACGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-742_AGACGA-JRDRM_L001_sorted.bam | gzip > COIJ42-742_AGACGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-742_AGACGA-JRDRM_L001_sorted.bam > COIJ42-742_AGACGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-742_AGACGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-742_AGACGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-742_AGACGA-JRDRM_L001.merged.fq_sorted.bam > COIJ42-742_AGACGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-742_AGACGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-742_AGACGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-742_AGACGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-742_AGACGA-JRDRM_L001.merged.fq_sorted.bam > COIJ42-742_AGACGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-742_AGACGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-743_GCTGCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-743_GCTGCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-743_GCTGCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-743_GCTGCC-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-743_GCTGCC-JRDRM_L001_sorted.bam > COIJ42-743_GCTGCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-743_GCTGCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-743_GCTGCC-JRDRM_L001_sorted.bam | gzip > COIJ42-743_GCTGCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-743_GCTGCC-JRDRM_L001_sorted.bam > COIJ42-743_GCTGCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-743_GCTGCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-743_GCTGCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-744_TAACCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-744_TAACCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-744_TAACCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-744_TAACCG-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-744_TAACCG-JRDRM_L001_sorted.bam > COIJ42-744_TAACCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-744_TAACCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-744_TAACCG-JRDRM_L001_sorted.bam | gzip > COIJ42-744_TAACCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-744_TAACCG-JRDRM_L001_sorted.bam > COIJ42-744_TAACCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-744_TAACCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-744_TAACCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-744_TAACCG-JRDRM_L001.merged.fq_sorted.bam > COIJ42-744_TAACCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-744_TAACCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-744_TAACCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-744_TAACCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-744_TAACCG-JRDRM_L001.merged.fq_sorted.bam > COIJ42-744_TAACCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-744_TAACCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-745_CTTCGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-745_CTTCGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-745_CTTCGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-745_CTTCGT-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-745_CTTCGT-JRDRM_L001_sorted.bam > COIJ42-745_CTTCGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-745_CTTCGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-745_CTTCGT-JRDRM_L001_sorted.bam | gzip > COIJ42-745_CTTCGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-745_CTTCGT-JRDRM_L001_sorted.bam > COIJ42-745_CTTCGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_sorted.bam > COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_sorted.bam > COIJ42-745_CTTCGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-745_CTTCGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-747_GTACAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-747_GTACAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-747_GTACAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-747_GTACAG-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-747_GTACAG-JRDRM_L001_sorted.bam > COIJ42-747_GTACAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-747_GTACAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-747_GTACAG-JRDRM_L001_sorted.bam | gzip > COIJ42-747_GTACAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-747_GTACAG-JRDRM_L001_sorted.bam > COIJ42-747_GTACAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-747_GTACAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-747_GTACAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-747_GTACAG-JRDRM_L001.merged.fq_sorted.bam > COIJ42-747_GTACAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-747_GTACAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-747_GTACAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-747_GTACAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-747_GTACAG-JRDRM_L001.merged.fq_sorted.bam > COIJ42-747_GTACAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-747_GTACAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-748_ATCACC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-748_ATCACC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-748_ATCACC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-748_ATCACC-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-748_ATCACC-JRDRM_L001_sorted.bam > COIJ42-748_ATCACC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-748_ATCACC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-748_ATCACC-JRDRM_L001_sorted.bam | gzip > COIJ42-748_ATCACC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-748_ATCACC-JRDRM_L001_sorted.bam > COIJ42-748_ATCACC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-748_ATCACC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-748_ATCACC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-748_ATCACC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-748_ATCACC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-748_ATCACC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-748_ATCACC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-748_ATCACC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-748_ATCACC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-748_ATCACC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-748_ATCACC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-749_TCCTGC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-749_TCCTGC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-749_TCCTGC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-749_TCCTGC-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-749_TCCTGC-JRDRM_L001_sorted.bam > COIJ42-749_TCCTGC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-749_TCCTGC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-749_TCCTGC-JRDRM_L001_sorted.bam | gzip > COIJ42-749_TCCTGC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-749_TCCTGC-JRDRM_L001_sorted.bam > COIJ42-749_TCCTGC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-749_TCCTGC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-749_TCCTGC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-752_TCGGCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-752_TCGGCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-752_TCGGCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-752_TCGGCC-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-752_TCGGCC-JRDRM_L001_sorted.bam > COIJ42-752_TCGGCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-752_TCGGCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-752_TCGGCC-JRDRM_L001_sorted.bam | gzip > COIJ42-752_TCGGCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-752_TCGGCC-JRDRM_L001_sorted.bam > COIJ42-752_TCGGCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_sorted.bam > COIJ42-752_TCGGCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-752_TCGGCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-753_GGTGAA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-753_GGTGAA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-753_GGTGAA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-753_GGTGAA-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-753_GGTGAA-JRDRM_L001_sorted.bam > COIJ42-753_GGTGAA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-753_GGTGAA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-753_GGTGAA-JRDRM_L001_sorted.bam | gzip > COIJ42-753_GGTGAA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-753_GGTGAA-JRDRM_L001_sorted.bam > COIJ42-753_GGTGAA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_sorted.bam > COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_sorted.bam > COIJ42-753_GGTGAA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-753_GGTGAA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-754_TTCCGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ42-754_TTCCGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-754_TTCCGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-754_TTCCGT-JRDRM_L001_sorted.bam; samtools flagstat COIJ42-754_TTCCGT-JRDRM_L001_sorted.bam > COIJ42-754_TTCCGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ42-754_TTCCGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ42-754_TTCCGT-JRDRM_L001_sorted.bam | gzip > COIJ42-754_TTCCGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-754_TTCCGT-JRDRM_L001_sorted.bam > COIJ42-754_TTCCGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_sorted.bam > COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_sorted.bam > COIJ42-754_TTCCGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ42-754_TTCCGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-10_CAGGAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-10_CAGGAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-10_CAGGAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-10_CAGGAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-10_CAGGAC-JRDRM_L001_sorted.bam > COIJ7-10_CAGGAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-10_CAGGAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-10_CAGGAC-JRDRM_L001_sorted.bam | gzip > COIJ7-10_CAGGAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-10_CAGGAC-JRDRM_L001_sorted.bam > COIJ7-10_CAGGAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_sorted.bam > COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_sorted.bam > COIJ7-10_CAGGAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-10_CAGGAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-11_ACCTCA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-11_ACCTCA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-11_ACCTCA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-11_ACCTCA-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-11_ACCTCA-JRDRM_L001_sorted.bam > COIJ7-11_ACCTCA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-11_ACCTCA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-11_ACCTCA-JRDRM_L001_sorted.bam | gzip > COIJ7-11_ACCTCA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-11_ACCTCA-JRDRM_L001_sorted.bam > COIJ7-11_ACCTCA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-11_ACCTCA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-11_ACCTCA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-12_AGGGAT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-12_AGGGAT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-12_AGGGAT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-12_AGGGAT-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-12_AGGGAT-JRDRM_L001_sorted.bam > COIJ7-12_AGGGAT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-12_AGGGAT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-12_AGGGAT-JRDRM_L001_sorted.bam | gzip > COIJ7-12_AGGGAT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-12_AGGGAT-JRDRM_L001_sorted.bam > COIJ7-12_AGGGAT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_sorted.bam > COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_sorted.bam > COIJ7-12_AGGGAT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-12_AGGGAT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-13_ACGATC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-13_ACGATC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-13_ACGATC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-13_ACGATC-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-13_ACGATC-JRDRM_L001_sorted.bam > COIJ7-13_ACGATC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-13_ACGATC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-13_ACGATC-JRDRM_L001_sorted.bam | gzip > COIJ7-13_ACGATC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-13_ACGATC-JRDRM_L001_sorted.bam > COIJ7-13_ACGATC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-13_ACGATC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-13_ACGATC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-13_ACGATC-JRDRM_L001.merged.fq_sorted.bam > COIJ7-13_ACGATC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-13_ACGATC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-13_ACGATC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-13_ACGATC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-13_ACGATC-JRDRM_L001.merged.fq_sorted.bam > COIJ7-13_ACGATC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-13_ACGATC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-14_AGGGCA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-14_AGGGCA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-14_AGGGCA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-14_AGGGCA-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-14_AGGGCA-JRDRM_L001_sorted.bam > COIJ7-14_AGGGCA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-14_AGGGCA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-14_AGGGCA-JRDRM_L001_sorted.bam | gzip > COIJ7-14_AGGGCA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-14_AGGGCA-JRDRM_L001_sorted.bam > COIJ7-14_AGGGCA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-14_AGGGCA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-14_AGGGCA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-15_CGATAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-15_CGATAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-15_CGATAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-15_CGATAC-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-15_CGATAC-JRDRM_L001_sorted.bam > COIJ7-15_CGATAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-15_CGATAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-15_CGATAC-JRDRM_L001_sorted.bam | gzip > COIJ7-15_CGATAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-15_CGATAC-JRDRM_L001_sorted.bam > COIJ7-15_CGATAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-15_CGATAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-15_CGATAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-15_CGATAC-JRDRM_L001.merged.fq_sorted.bam > COIJ7-15_CGATAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-15_CGATAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-15_CGATAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-15_CGATAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-15_CGATAC-JRDRM_L001.merged.fq_sorted.bam > COIJ7-15_CGATAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-15_CGATAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-17_TCCTTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-17_TCCTTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-17_TCCTTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-17_TCCTTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-17_TCCTTA-JRDRM_L001_sorted.bam > COIJ7-17_TCCTTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-17_TCCTTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-17_TCCTTA-JRDRM_L001_sorted.bam | gzip > COIJ7-17_TCCTTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-17_TCCTTA-JRDRM_L001_sorted.bam > COIJ7-17_TCCTTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-17_TCCTTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-17_TCCTTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-19_GAGTTG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-19_GAGTTG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-19_GAGTTG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-19_GAGTTG-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-19_GAGTTG-JRDRM_L001_sorted.bam > COIJ7-19_GAGTTG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-19_GAGTTG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-19_GAGTTG-JRDRM_L001_sorted.bam | gzip > COIJ7-19_GAGTTG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-19_GAGTTG-JRDRM_L001_sorted.bam > COIJ7-19_GAGTTG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-19_GAGTTG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-19_GAGTTG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-21_AATGAA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-21_AATGAA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-21_AATGAA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-21_AATGAA-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-21_AATGAA-JRDRM_L001_sorted.bam > COIJ7-21_AATGAA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-21_AATGAA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-21_AATGAA-JRDRM_L001_sorted.bam | gzip > COIJ7-21_AATGAA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-21_AATGAA-JRDRM_L001_sorted.bam > COIJ7-21_AATGAA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-21_AATGAA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-21_AATGAA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-21_AATGAA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-21_AATGAA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-21_AATGAA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-21_AATGAA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-21_AATGAA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-21_AATGAA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-21_AATGAA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-21_AATGAA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-22_TGTGAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-22_TGTGAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-22_TGTGAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-22_TGTGAG-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-22_TGTGAG-JRDRM_L001_sorted.bam > COIJ7-22_TGTGAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-22_TGTGAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-22_TGTGAG-JRDRM_L001_sorted.bam | gzip > COIJ7-22_TGTGAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-22_TGTGAG-JRDRM_L001_sorted.bam > COIJ7-22_TGTGAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-22_TGTGAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-22_TGTGAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-2_TCGAGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-2_TCGAGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-2_TCGAGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-2_TCGAGG-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-2_TCGAGG-JRDRM_L001_sorted.bam > COIJ7-2_TCGAGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-2_TCGAGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-2_TCGAGG-JRDRM_L001_sorted.bam | gzip > COIJ7-2_TCGAGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-2_TCGAGG-JRDRM_L001_sorted.bam > COIJ7-2_TCGAGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-2_TCGAGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-2_TCGAGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-3_AGTAAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-3_AGTAAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-3_AGTAAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-3_AGTAAG-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-3_AGTAAG-JRDRM_L001_sorted.bam > COIJ7-3_AGTAAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-3_AGTAAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-3_AGTAAG-JRDRM_L001_sorted.bam | gzip > COIJ7-3_AGTAAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-3_AGTAAG-JRDRM_L001_sorted.bam > COIJ7-3_AGTAAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-3_AGTAAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-3_AGTAAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-4_CTCATA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-4_CTCATA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-4_CTCATA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-4_CTCATA-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-4_CTCATA-JRDRM_L001_sorted.bam > COIJ7-4_CTCATA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-4_CTCATA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-4_CTCATA-JRDRM_L001_sorted.bam | gzip > COIJ7-4_CTCATA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-4_CTCATA-JRDRM_L001_sorted.bam > COIJ7-4_CTCATA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-4_CTCATA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-4_CTCATA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-4_CTCATA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-4_CTCATA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-4_CTCATA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-4_CTCATA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-4_CTCATA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-4_CTCATA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-4_CTCATA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-4_CTCATA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-6_TGCAGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-6_TGCAGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-6_TGCAGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-6_TGCAGT-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-6_TGCAGT-JRDRM_L001_sorted.bam > COIJ7-6_TGCAGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-6_TGCAGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-6_TGCAGT-JRDRM_L001_sorted.bam | gzip > COIJ7-6_TGCAGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-6_TGCAGT-JRDRM_L001_sorted.bam > COIJ7-6_TGCAGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_sorted.bam > COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_sorted.bam > COIJ7-6_TGCAGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-6_TGCAGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-7_AACTTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-7_AACTTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-7_AACTTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-7_AACTTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-7_AACTTA-JRDRM_L001_sorted.bam > COIJ7-7_AACTTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-7_AACTTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-7_AACTTA-JRDRM_L001_sorted.bam | gzip > COIJ7-7_AACTTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-7_AACTTA-JRDRM_L001_sorted.bam > COIJ7-7_AACTTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-7_AACTTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-7_AACTTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-7_AACTTA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-7_AACTTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-7_AACTTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-7_AACTTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-7_AACTTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-7_AACTTA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-7_AACTTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-7_AACTTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-8_GCATGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-8_GCATGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-8_GCATGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-8_GCATGG-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-8_GCATGG-JRDRM_L001_sorted.bam > COIJ7-8_GCATGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-8_GCATGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-8_GCATGG-JRDRM_L001_sorted.bam | gzip > COIJ7-8_GCATGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-8_GCATGG-JRDRM_L001_sorted.bam > COIJ7-8_GCATGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-8_GCATGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-8_GCATGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-8_GCATGG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-8_GCATGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-8_GCATGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-8_GCATGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-8_GCATGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-8_GCATGG-JRDRM_L001.merged.fq_sorted.bam > COIJ7-8_GCATGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-8_GCATGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-9_GTCCTA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIJ7-9_GTCCTA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-9_GTCCTA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-9_GTCCTA-JRDRM_L001_sorted.bam; samtools flagstat COIJ7-9_GTCCTA-JRDRM_L001_sorted.bam > COIJ7-9_GTCCTA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIJ7-9_GTCCTA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIJ7-9_GTCCTA-JRDRM_L001_sorted.bam | gzip > COIJ7-9_GTCCTA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-9_GTCCTA-JRDRM_L001_sorted.bam > COIJ7-9_GTCCTA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_sorted.bam > COIJ7-9_GTCCTA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIJ7-9_GTCCTA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-646_GTGCCA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIL3-646_GTGCCA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-646_GTGCCA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-646_GTGCCA-JRDRM_L001_sorted.bam; samtools flagstat COIL3-646_GTGCCA-JRDRM_L001_sorted.bam > COIL3-646_GTGCCA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIL3-646_GTGCCA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIL3-646_GTGCCA-JRDRM_L001_sorted.bam | gzip > COIL3-646_GTGCCA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-646_GTGCCA-JRDRM_L001_sorted.bam > COIL3-646_GTGCCA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-646_GTGCCA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-646_GTGCCA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIL3-646_GTGCCA-JRDRM_L001.merged.fq_sorted.bam > COIL3-646_GTGCCA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIL3-646_GTGCCA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIL3-646_GTGCCA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIL3-646_GTGCCA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-646_GTGCCA-JRDRM_L001.merged.fq_sorted.bam > COIL3-646_GTGCCA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIL3-646_GTGCCA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-710_TTCAAC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIL3-710_TTCAAC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-710_TTCAAC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-710_TTCAAC-JRDRM_L001_sorted.bam; samtools flagstat COIL3-710_TTCAAC-JRDRM_L001_sorted.bam > COIL3-710_TTCAAC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIL3-710_TTCAAC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIL3-710_TTCAAC-JRDRM_L001_sorted.bam | gzip > COIL3-710_TTCAAC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-710_TTCAAC-JRDRM_L001_sorted.bam > COIL3-710_TTCAAC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-710_TTCAAC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-710_TTCAAC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIL3-710_TTCAAC-JRDRM_L001.merged.fq_sorted.bam > COIL3-710_TTCAAC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIL3-710_TTCAAC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIL3-710_TTCAAC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIL3-710_TTCAAC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-710_TTCAAC-JRDRM_L001.merged.fq_sorted.bam > COIL3-710_TTCAAC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIL3-710_TTCAAC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-717_AATGGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIL3-717_AATGGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-717_AATGGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-717_AATGGT-JRDRM_L001_sorted.bam; samtools flagstat COIL3-717_AATGGT-JRDRM_L001_sorted.bam > COIL3-717_AATGGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIL3-717_AATGGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIL3-717_AATGGT-JRDRM_L001_sorted.bam | gzip > COIL3-717_AATGGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-717_AATGGT-JRDRM_L001_sorted.bam > COIL3-717_AATGGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-717_AATGGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-717_AATGGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIL3-717_AATGGT-JRDRM_L001.merged.fq_sorted.bam > COIL3-717_AATGGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIL3-717_AATGGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIL3-717_AATGGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIL3-717_AATGGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-717_AATGGT-JRDRM_L001.merged.fq_sorted.bam > COIL3-717_AATGGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIL3-717_AATGGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM1-1_AGCCCT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIM1-1_AGCCCT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM1-1_AGCCCT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM1-1_AGCCCT-JRDRM_L001_sorted.bam; samtools flagstat COIM1-1_AGCCCT-JRDRM_L001_sorted.bam > COIM1-1_AGCCCT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIM1-1_AGCCCT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIM1-1_AGCCCT-JRDRM_L001_sorted.bam | gzip > COIM1-1_AGCCCT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIM1-1_AGCCCT-JRDRM_L001_sorted.bam > COIM1-1_AGCCCT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM1-1_AGCCCT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM1-1_AGCCCT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIM1-1_AGCCCT-JRDRM_L001.merged.fq_sorted.bam > COIM1-1_AGCCCT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM1-1_AGCCCT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM1-1_AGCCCT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIM1-1_AGCCCT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM1-1_AGCCCT-JRDRM_L001.merged.fq_sorted.bam > COIM1-1_AGCCCT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM1-1_AGCCCT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM1_TGGCCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIM1_TGGCCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM1_TGGCCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM1_TGGCCG-JRDRM_L001_sorted.bam; samtools flagstat COIM1_TGGCCG-JRDRM_L001_sorted.bam > COIM1_TGGCCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIM1_TGGCCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIM1_TGGCCG-JRDRM_L001_sorted.bam | gzip > COIM1_TGGCCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIM1_TGGCCG-JRDRM_L001_sorted.bam > COIM1_TGGCCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM1_TGGCCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM1_TGGCCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIM1_TGGCCG-JRDRM_L001.merged.fq_sorted.bam > COIM1_TGGCCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM1_TGGCCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM1_TGGCCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIM1_TGGCCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM1_TGGCCG-JRDRM_L001.merged.fq_sorted.bam > COIM1_TGGCCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM1_TGGCCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM2-1_CCAGCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIM2-1_CCAGCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM2-1_CCAGCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM2-1_CCAGCC-JRDRM_L001_sorted.bam; samtools flagstat COIM2-1_CCAGCC-JRDRM_L001_sorted.bam > COIM2-1_CCAGCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIM2-1_CCAGCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIM2-1_CCAGCC-JRDRM_L001_sorted.bam | gzip > COIM2-1_CCAGCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIM2-1_CCAGCC-JRDRM_L001_sorted.bam > COIM2-1_CCAGCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM2-1_CCAGCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM2-1_CCAGCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIM2-1_CCAGCC-JRDRM_L001.merged.fq_sorted.bam > COIM2-1_CCAGCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM2-1_CCAGCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM2-1_CCAGCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIM2-1_CCAGCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM2-1_CCAGCC-JRDRM_L001.merged.fq_sorted.bam > COIM2-1_CCAGCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM2-1_CCAGCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM2_CATGAG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIM2_CATGAG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM2_CATGAG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM2_CATGAG-JRDRM_L001_sorted.bam; samtools flagstat COIM2_CATGAG-JRDRM_L001_sorted.bam > COIM2_CATGAG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIM2_CATGAG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIM2_CATGAG-JRDRM_L001_sorted.bam | gzip > COIM2_CATGAG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIM2_CATGAG-JRDRM_L001_sorted.bam > COIM2_CATGAG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM2_CATGAG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM2_CATGAG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIM2_CATGAG-JRDRM_L001.merged.fq_sorted.bam > COIM2_CATGAG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM2_CATGAG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM2_CATGAG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIM2_CATGAG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM2_CATGAG-JRDRM_L001.merged.fq_sorted.bam > COIM2_CATGAG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM2_CATGAG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM3_CCCTCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIM3_CCCTCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM3_CCCTCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM3_CCCTCC-JRDRM_L001_sorted.bam; samtools flagstat COIM3_CCCTCC-JRDRM_L001_sorted.bam > COIM3_CCCTCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIM3_CCCTCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIM3_CCCTCC-JRDRM_L001_sorted.bam | gzip > COIM3_CCCTCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIM3_CCCTCC-JRDRM_L001_sorted.bam > COIM3_CCCTCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM3_CCCTCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM3_CCCTCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIM3_CCCTCC-JRDRM_L001.merged.fq_sorted.bam > COIM3_CCCTCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM3_CCCTCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM3_CCCTCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIM3_CCCTCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM3_CCCTCC-JRDRM_L001.merged.fq_sorted.bam > COIM3_CCCTCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM3_CCCTCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM4_TGTTCG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIM4_TGTTCG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM4_TGTTCG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM4_TGTTCG-JRDRM_L001_sorted.bam; samtools flagstat COIM4_TGTTCG-JRDRM_L001_sorted.bam > COIM4_TGTTCG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIM4_TGTTCG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIM4_TGTTCG-JRDRM_L001_sorted.bam | gzip > COIM4_TGTTCG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIM4_TGTTCG-JRDRM_L001_sorted.bam > COIM4_TGTTCG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM4_TGTTCG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM4_TGTTCG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIM4_TGTTCG-JRDRM_L001.merged.fq_sorted.bam > COIM4_TGTTCG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM4_TGTTCG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM4_TGTTCG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIM4_TGTTCG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM4_TGTTCG-JRDRM_L001.merged.fq_sorted.bam > COIM4_TGTTCG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM4_TGTTCG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM5_CTAGTC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIM5_CTAGTC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM5_CTAGTC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM5_CTAGTC-JRDRM_L001_sorted.bam; samtools flagstat COIM5_CTAGTC-JRDRM_L001_sorted.bam > COIM5_CTAGTC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIM5_CTAGTC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIM5_CTAGTC-JRDRM_L001_sorted.bam | gzip > COIM5_CTAGTC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIM5_CTAGTC-JRDRM_L001_sorted.bam > COIM5_CTAGTC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM5_CTAGTC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM5_CTAGTC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIM5_CTAGTC-JRDRM_L001.merged.fq_sorted.bam > COIM5_CTAGTC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM5_CTAGTC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM5_CTAGTC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIM5_CTAGTC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM5_CTAGTC-JRDRM_L001.merged.fq_sorted.bam > COIM5_CTAGTC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIM5_CTAGTC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMeq_AAGTGG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIMeq_AAGTGG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMeq_AAGTGG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMeq_AAGTGG-JRDRM_L001_sorted.bam; samtools flagstat COIMeq_AAGTGG-JRDRM_L001_sorted.bam > COIMeq_AAGTGG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIMeq_AAGTGG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIMeq_AAGTGG-JRDRM_L001_sorted.bam | gzip > COIMeq_AAGTGG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIMeq_AAGTGG-JRDRM_L001_sorted.bam > COIMeq_AAGTGG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMeq_AAGTGG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMeq_AAGTGG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIMeq_AAGTGG-JRDRM_L001.merged.fq_sorted.bam > COIMeq_AAGTGG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIMeq_AAGTGG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIMeq_AAGTGG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIMeq_AAGTGG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIMeq_AAGTGG-JRDRM_L001.merged.fq_sorted.bam > COIMeq_AAGTGG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIMeq_AAGTGG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMeq-C12_TTGTGA-JG62T_L001_R1.fastq -relabel @ -fastqout ./COIMeq-C12_TTGTGA-JG62T_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMeq-C12_TTGTGA-JG62T_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMeq-C12_TTGTGA-JG62T_L001_sorted.bam; samtools flagstat COIMeq-C12_TTGTGA-JG62T_L001_sorted.bam > COIMeq-C12_TTGTGA-JG62T_L001_sorted.bam.flagstat.txt ; samtools index COIMeq-C12_TTGTGA-JG62T_L001_sorted.bam ; bedtools genomecov -d -ibam COIMeq-C12_TTGTGA-JG62T_L001_sorted.bam | gzip > COIMeq-C12_TTGTGA-JG62T_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIMeq-C12_TTGTGA-JG62T_L001_sorted.bam > COIMeq-C12_TTGTGA-JG62T_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_sorted.bam; samtools flagstat COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_sorted.bam > COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_sorted.bam | gzip > COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_sorted.bam > COIMeq-C12_TTGTGA-JG62T_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_sorted.bam > COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_sorted.bam | gzip > COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_sorted.bam > COIMeq-C12_TTGTGA-JG62T_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMS10-C12_CCCCAT-JG62T_L001_R1.fastq -relabel @ -fastqout ./COIMS10-C12_CCCCAT-JG62T_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMS10-C12_CCCCAT-JG62T_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMS10-C12_CCCCAT-JG62T_L001_sorted.bam; samtools flagstat COIMS10-C12_CCCCAT-JG62T_L001_sorted.bam > COIMS10-C12_CCCCAT-JG62T_L001_sorted.bam.flagstat.txt ; samtools index COIMS10-C12_CCCCAT-JG62T_L001_sorted.bam ; bedtools genomecov -d -ibam COIMS10-C12_CCCCAT-JG62T_L001_sorted.bam | gzip > COIMS10-C12_CCCCAT-JG62T_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIMS10-C12_CCCCAT-JG62T_L001_sorted.bam > COIMS10-C12_CCCCAT-JG62T_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_sorted.bam; samtools flagstat COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_sorted.bam > COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_sorted.bam | gzip > COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_sorted.bam > COIMS10-C12_CCCCAT-JG62T_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_sorted.bam > COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_sorted.bam | gzip > COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_sorted.bam > COIMS10-C12_CCCCAT-JG62T_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMS10_GCCACA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIMS10_GCCACA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMS10_GCCACA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMS10_GCCACA-JRDRM_L001_sorted.bam; samtools flagstat COIMS10_GCCACA-JRDRM_L001_sorted.bam > COIMS10_GCCACA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIMS10_GCCACA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIMS10_GCCACA-JRDRM_L001_sorted.bam | gzip > COIMS10_GCCACA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIMS10_GCCACA-JRDRM_L001_sorted.bam > COIMS10_GCCACA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMS10_GCCACA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMS10_GCCACA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIMS10_GCCACA-JRDRM_L001.merged.fq_sorted.bam > COIMS10_GCCACA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIMS10_GCCACA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIMS10_GCCACA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIMS10_GCCACA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIMS10_GCCACA-JRDRM_L001.merged.fq_sorted.bam > COIMS10_GCCACA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIMS10_GCCACA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-646_AGTATT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-646_AGTATT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-646_AGTATT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-646_AGTATT-JRDRM_L001_sorted.bam; samtools flagstat COIPool-646_AGTATT-JRDRM_L001_sorted.bam > COIPool-646_AGTATT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-646_AGTATT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-646_AGTATT-JRDRM_L001_sorted.bam | gzip > COIPool-646_AGTATT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-646_AGTATT-JRDRM_L001_sorted.bam > COIPool-646_AGTATT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-646_AGTATT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-646_AGTATT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-646_AGTATT-JRDRM_L001.merged.fq_sorted.bam > COIPool-646_AGTATT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-646_AGTATT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-646_AGTATT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-646_AGTATT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-646_AGTATT-JRDRM_L001.merged.fq_sorted.bam > COIPool-646_AGTATT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-646_AGTATT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-710_GTTAGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-710_GTTAGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-710_GTTAGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-710_GTTAGT-JRDRM_L001_sorted.bam; samtools flagstat COIPool-710_GTTAGT-JRDRM_L001_sorted.bam > COIPool-710_GTTAGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-710_GTTAGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-710_GTTAGT-JRDRM_L001_sorted.bam | gzip > COIPool-710_GTTAGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-710_GTTAGT-JRDRM_L001_sorted.bam > COIPool-710_GTTAGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-710_GTTAGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-710_GTTAGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-710_GTTAGT-JRDRM_L001.merged.fq_sorted.bam > COIPool-710_GTTAGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-710_GTTAGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-710_GTTAGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-710_GTTAGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-710_GTTAGT-JRDRM_L001.merged.fq_sorted.bam > COIPool-710_GTTAGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-710_GTTAGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-717_GTGCTG-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-717_GTGCTG-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-717_GTGCTG-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-717_GTGCTG-JRDRM_L001_sorted.bam; samtools flagstat COIPool-717_GTGCTG-JRDRM_L001_sorted.bam > COIPool-717_GTGCTG-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-717_GTGCTG-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-717_GTGCTG-JRDRM_L001_sorted.bam | gzip > COIPool-717_GTGCTG-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-717_GTGCTG-JRDRM_L001_sorted.bam > COIPool-717_GTGCTG-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-717_GTGCTG-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-717_GTGCTG-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-717_GTGCTG-JRDRM_L001.merged.fq_sorted.bam > COIPool-717_GTGCTG-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-717_GTGCTG-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-717_GTGCTG-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-717_GTGCTG-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-717_GTGCTG-JRDRM_L001.merged.fq_sorted.bam > COIPool-717_GTGCTG-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-717_GTGCTG-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-726_CAATGC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-726_CAATGC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-726_CAATGC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-726_CAATGC-JRDRM_L001_sorted.bam; samtools flagstat COIPool-726_CAATGC-JRDRM_L001_sorted.bam > COIPool-726_CAATGC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-726_CAATGC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-726_CAATGC-JRDRM_L001_sorted.bam | gzip > COIPool-726_CAATGC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-726_CAATGC-JRDRM_L001_sorted.bam > COIPool-726_CAATGC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-726_CAATGC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-726_CAATGC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-726_CAATGC-JRDRM_L001.merged.fq_sorted.bam > COIPool-726_CAATGC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-726_CAATGC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-726_CAATGC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-726_CAATGC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-726_CAATGC-JRDRM_L001.merged.fq_sorted.bam > COIPool-726_CAATGC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-726_CAATGC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-729_TATCAT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-729_TATCAT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-729_TATCAT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-729_TATCAT-JRDRM_L001_sorted.bam; samtools flagstat COIPool-729_TATCAT-JRDRM_L001_sorted.bam > COIPool-729_TATCAT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-729_TATCAT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-729_TATCAT-JRDRM_L001_sorted.bam | gzip > COIPool-729_TATCAT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-729_TATCAT-JRDRM_L001_sorted.bam > COIPool-729_TATCAT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-729_TATCAT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-729_TATCAT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-729_TATCAT-JRDRM_L001.merged.fq_sorted.bam > COIPool-729_TATCAT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-729_TATCAT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-729_TATCAT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-729_TATCAT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-729_TATCAT-JRDRM_L001.merged.fq_sorted.bam > COIPool-729_TATCAT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-729_TATCAT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-733_CGCGCC-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-733_CGCGCC-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-733_CGCGCC-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-733_CGCGCC-JRDRM_L001_sorted.bam; samtools flagstat COIPool-733_CGCGCC-JRDRM_L001_sorted.bam > COIPool-733_CGCGCC-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-733_CGCGCC-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-733_CGCGCC-JRDRM_L001_sorted.bam | gzip > COIPool-733_CGCGCC-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-733_CGCGCC-JRDRM_L001_sorted.bam > COIPool-733_CGCGCC-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-733_CGCGCC-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-733_CGCGCC-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-733_CGCGCC-JRDRM_L001.merged.fq_sorted.bam > COIPool-733_CGCGCC-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-733_CGCGCC-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-733_CGCGCC-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-733_CGCGCC-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-733_CGCGCC-JRDRM_L001.merged.fq_sorted.bam > COIPool-733_CGCGCC-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-733_CGCGCC-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-734_TCATGT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-734_TCATGT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-734_TCATGT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-734_TCATGT-JRDRM_L001_sorted.bam; samtools flagstat COIPool-734_TCATGT-JRDRM_L001_sorted.bam > COIPool-734_TCATGT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-734_TCATGT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-734_TCATGT-JRDRM_L001_sorted.bam | gzip > COIPool-734_TCATGT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-734_TCATGT-JRDRM_L001_sorted.bam > COIPool-734_TCATGT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-734_TCATGT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-734_TCATGT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-734_TCATGT-JRDRM_L001.merged.fq_sorted.bam > COIPool-734_TCATGT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-734_TCATGT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-734_TCATGT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-734_TCATGT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-734_TCATGT-JRDRM_L001.merged.fq_sorted.bam > COIPool-734_TCATGT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-734_TCATGT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-748_CGGCAT-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-748_CGGCAT-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-748_CGGCAT-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-748_CGGCAT-JRDRM_L001_sorted.bam; samtools flagstat COIPool-748_CGGCAT-JRDRM_L001_sorted.bam > COIPool-748_CGGCAT-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-748_CGGCAT-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-748_CGGCAT-JRDRM_L001_sorted.bam | gzip > COIPool-748_CGGCAT-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-748_CGGCAT-JRDRM_L001_sorted.bam > COIPool-748_CGGCAT-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-748_CGGCAT-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-748_CGGCAT-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-748_CGGCAT-JRDRM_L001.merged.fq_sorted.bam > COIPool-748_CGGCAT-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-748_CGGCAT-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-748_CGGCAT-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-748_CGGCAT-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-748_CGGCAT-JRDRM_L001.merged.fq_sorted.bam > COIPool-748_CGGCAT-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-748_CGGCAT-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-755_GGAAGA-JRDRM_L001_R1.fastq -relabel @ -fastqout ./COIPool-755_GGAAGA-JRDRM_L001.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-755_GGAAGA-JRDRM_L001.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-755_GGAAGA-JRDRM_L001_sorted.bam; samtools flagstat COIPool-755_GGAAGA-JRDRM_L001_sorted.bam > COIPool-755_GGAAGA-JRDRM_L001_sorted.bam.flagstat.txt ; samtools index COIPool-755_GGAAGA-JRDRM_L001_sorted.bam ; bedtools genomecov -d -ibam COIPool-755_GGAAGA-JRDRM_L001_sorted.bam | gzip > COIPool-755_GGAAGA-JRDRM_L001_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-755_GGAAGA-JRDRM_L001_sorted.bam > COIPool-755_GGAAGA-JRDRM_L001_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-755_GGAAGA-JRDRM_L001.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-755_GGAAGA-JRDRM_L001.merged.fq_sorted.bam; samtools flagstat COIPool-755_GGAAGA-JRDRM_L001.merged.fq_sorted.bam > COIPool-755_GGAAGA-JRDRM_L001.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-755_GGAAGA-JRDRM_L001.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-755_GGAAGA-JRDRM_L001.merged.fq_sorted.bam | gzip > COIPool-755_GGAAGA-JRDRM_L001.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-755_GGAAGA-JRDRM_L001.merged.fq_sorted.bam > COIPool-755_GGAAGA-JRDRM_L001.merged.fq_idxstats.txt
module load bioinfo/bedtools2-2.29.0; module load bioinfo/usearch11.0.667; module load bioinfo/minimap2-2.11; module load bioinfo/samtools-1.10; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/; usearch -fastq_mergepairs ./COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_R1.fastq -relabel @ -fastqout ./COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq.merged.fq ; cd /work/project/isp/MPN/CYATHOMIX/NMB_XP/mock_MINIMAPm_clus99_B4_k10_w10; minimap2 -t 4 -ax sr -A2 -B4 -O4,24 -E2,1 -k10 -w10 /work/project/isp/MPN/CYATHOMIX/NMB_XP/all_cox1_concat_noeleph_rmdup_clust99.fasta /work/project/isp/MPN/CYATHOMIX/NMB_XP/cutadapt/COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq.merged.fq | samtools view -b -F 2308 -q 0| samtools sort -@ 4 - -o COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam; samtools flagstat COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam.flagstat.txt ; samtools index COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam ; bedtools genomecov -d -ibam COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam | gzip > COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_nofilt_genomecov_d.txt.gz; samtools idxstats COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_sorted.bam > COIPool-755_GGAAGA-JRDRM_L001.merged.fq.merged.fq_idxstats.txt
